package/yast2-registration.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Fri Feb 12 12:42:19 UTC 2016 - knut.anderssen@suse.com
+
+- Continue asking for registration codes also after failed
```

Comment:
I would rephrase it. Maybe something like: `Ask again when an incorrect registration code is provided`.